packages/itwin/ec3-widget/src/components/api/EC3ConfigurationClient.ts

```diff
@@ -0,0 +1,141 @@
+/*---------------------------------------------------------------------------------------------
+* Copyright (c) Bentley Systems, Incorporated. All rights reserved.
+* See LICENSE.md in the project root for license terms and full copyright notice.
+*--------------------------------------------------------------------------------------------*/
+import { IModelApp } from "@itwin/core-frontend";
+import type { Configuration } from "../Template";
+import isomorphicFetch from "cross-fetch";
+
+const ACCEPT = "application/vnd.bentley.itwin-platform.v1+json";
+
+export const BASE_PATH = "https://api.bentley.com/insights/carbon-calculation/ec3";
+
+class EC3ConfigurationClient {
```

Comment:
Move EC3 clients to `insights-clients` npm pacakge.

Reply:
PR is open